data/coalition/coalition missions.txt

```diff
@@ -1393,3 +1393,63 @@ mission "Arachi Cartoonist 2"
+		conversation
+			`You walk around the spaceport for a few minutes, when suddenly you're tackled into the ground, with whoever attacked you pinning you down, holding your arms behind your back to restrain you. When you manage to look back, you see a man of human appearance, with very long hair and wearing an intricate necklace of gold. His most notable trait, however, is his green skin - an Alpha.`
+			`	Your eyes don't meet for long, though, as he looks to his side and lets go of your arms to try and block an Arach that swoops in with a tackle, knocking him off of you. They wrestle for a while, as you get up, but it doesn't take long for the Arach Heliarch to restrain the Alpha, with all eight legs firmly holding his arms to his torso, as the Arach rests her entire body weight on the Alpha's back. A Saryd Heliarch approaches you, telling you that "Keep your distance, you must," and then looking at the Alpha. "What reason for attacking this person, did you have?"`
+			`	The Alpha, still struggling to try and get out of the Arach's grip, ignores the question, looking at you instead, and asking, "You're with the Navy, aren't you? Finally figured out I was here?"`
+			branch navy
+				has "license: Navy"
+			choice
+				`	"No, I'm not with the Navy. You could've just asked without slamming me to the ground."`
+					goto notnavy
+				`	"Is that why you attacked me?"`
+					goto detained
+			label navy
+			choice
+				`	"I am with the Navy, but I didn't come here looking for you."`
+				`	"Is that why you attacked me?"`
+					goto detained
+			`	"And I'm supposed to believe that crap?" He lashes out, trying to knock the Arach off of him, but the agent only tightens the grip, and opens her mandibles, getting them close to the Alpha's neck. "Alright, alright, I get it!" The Alpha shouts, stopping his attempts to break free.`
+				goto detained
+			label notnavy
+			`	He stares at you for a moment, as if unsure if you're telling the truth. "What the hell are you doing here then? Did you just stumble onto this place?" He stops struggling, but the Arach agent doesn't loosen her grip.`
+			label detained
+			`	The Saryd walks closer to the Alpha, stomping his hoofs on the ground with each step. "Whatever your motivation was, a transgression, you have committed. Punished accordingly, you shall be." He says. "Kindly follow us to the detention facility, would you, Captain?" He asks you, but doesn't wait for an answer, as more agents arrive to escort you and the Alpha to the "detention facility."`
+			`	It's a short walk, and you end up in a building similar to a police station. The Alpha is brought before a Kimek that you guess is in charge of the facility. He asks you to give a testimony of what happened, with the Heliarch agents confirming what you say. "Your first offense in a long time this is, Alitis," the Kimek says, after checking a computer for the Alpha's data. "Three years in seclusion, your sentence is."`
```

Comment:
Is this intentionally intended to be authoritarian and harsh? Also, is punishment time species dependent? This seems like it would be an appreciable chunk of a Kimek's lifespan.

Reply:
Yes. No. The chunk of time I picked didn't have to do with species or anything, I just wanted a long time for the comparatively minor offense, if three years is too little for an Alpha could increase it.

Reply:
There's no point increasing it if the answer to the second question is "No" - that just makes minor infractions nearly equivalent to capital punishment for Kimek.